k8s/templates/business-create-ui.dc-srvc-rt-cmap-kcmap.json

```diff
@@ -276,63 +276,63 @@
             "name": "DOMAIN",
             "displayName": "Domain name.",
             "description": "The FQDN of the service.",
-            "required": false,
+            "required": true,
             "value": "business-create-dev.pathfinder.gov.bc.ca"
         },
         {
             "name": "PATH",
             "displayName": "URL Path",
             "description": "The path portion of the URL",
-            "required": false,
+            "required": true,
             "value": "/businesses/create/"
         },
         {
             "name": "LEGAL_API_URL",
             "displayName": "LEGAL API URL",
             "description": "The URL to the Legal API.",
-            "required": false,
+            "required": true,
             "value": ""
         },
         {
             "name": "AUTH_API_URL",
             "displayName": "Authorization API URL",
             "description": "The URL to the authorization service API.",
-            "required": false,
+            "required": true,
             "value": ""
         },
         {
             "name": "BUSINESSES_URL",
             "displayName": "Businesses Registry URL",
             "description": "The URL to the Business Registry home page.",
-            "required": false,
+            "required": true,
             "value": ""
         },
         {
             "name": "AUTH_URL",
             "displayName": "Auth URL",
             "description": "The URL to authenticate with entities.",
-            "required": false,
+            "required": true,
             "value": ""
         },
         {
             "name": "DASHBOARD_URL",
             "displayName": "Business Dashboard URL",
             "description": "The URL to the Business Dashboard (Filings UI).",
-            "required": false,
+            "required": true,
             "value": ""
         },
         {
             "name": "PAY_API_URL",
             "displayName": "Payment API URL",
             "description": "The URL to the payment API.",
-            "required": false,
+            "required": true,
             "value": ""
         },
         {
-            "name": "REPORT_SVC_URL",
-            "displayName": "Report Service API URL",
-            "description": "The URL to the Report Service API.",
-            "required": false,
+            "name": "SBC_CONFIG_AUTH_API_URL",
+            "displayName": "Auth URL",
+            "description": "The uth URL with no slash.",
```

Comment:
Just a typo in the description. 

Also, I thought I had already made this change in another ticket but I must have accidentally reset it locally and it didn't make it into my PR. So thanks for catching that!